nservicebus/upgrades/sqlserver_2to3.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Upgrade from Version 5 to Version 6
+summary: Instructions on how to upgrade from NServiceBus Versions 5 to 6
+tags:
+ - upgrade
+ - migration
+related:
+- nservicebus/sagas/concurrency
+---
+
+
+## SQL Server Transport
+
+### Transactions
+Transaction settings are now handled in the transport level configuration. For details refer to [Transaction configuraton API](/nservicebus/upgrades/5to6.md#transaction-configuration-API) section.
```

Comment:
@weralabaj - Not saying we should repeat content. You can always link to more details, but the first statement seems unclear. What are we trying to highlight to the user? Maybe we can come up with something better.  

Are we trying to say that with NServiceBus Versions 6 and above, the transaction API has a changed. And how it impacts the new version of SQL Server transport and the transaction scope settings?